__mocks__/globby.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * __mocks__/globby.js
+ * Mock globbing a directory
+ */
+'use strict';
+
+const q = require('q');
```

Comment:
where is this package being installed / defined as a dep? can't find it in any `package.json`

Reply:
it's an implementation of promises in js, is it? why use this instead of plain old es6 promises? these test files are already written in es6

Reply:
it is here: https://github.com/springernature/frontend-package-manager/blob/2959b2e60d8513b9b220a75a6a7bdd561d329d19/package.json#L39

Reply:
... but that is a good point! I copied the functionality over from the glob mock, so maybe this was a hangover from the old way of writing the package. I will take a look

Reply:
Yep, have got rid of both instances of this